server-api/src/scenario/TestScenarioFactory.ts

```
@@ -63,63 +63,65 @@ export class TestScenarioFactory {
       baseScenario.scenarioSetupSucceeded = true;
-    } catch (e) {
-      LogManager.getLogger().error(
-        `Unable to create core scenario setup: ${e}`
+
+      LogManager.getLogger().info('Initial base scenario setup created');
+      if (!scenarioConfig.space) {
+        // nothing more to do, return
+        return baseScenario;
+      }
+
+      baseScenario.space = await this.createRootSpace(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)